tests/common-tests.h

```diff
@@ -288,8 +348,43 @@ template<> struct type_conversion<PhonebookEntry3>
     }
 };
 
+template<typename T>
+struct type_conversion<DdlTester<T>>
+{
+    typedef soci::values base_type;
+    static void from_base(soci::values const &v, soci::indicator, DdlTester<T>& t)
+    {
+        map_ddl_tester(t, v);
+    }
+    static void to_base(DdlTester<T> const &t, soci::values &v, soci::indicator&)
+    {
+        v.set("VAL", t.inVal);
+    }
+};
 } // namespace soci
 
+template<typename T>
+void test_limit(soci::session &sql, DdlTester<T>&& tester)
```

Comment:
The name of this function is confusing, why does it say "limit" when there are no limits in play here? Something like `test_roundtrip()` would be more clear.